rocAL/include/pipeline/tensor.h

```diff
@@ -200,14 +200,12 @@ class TensorInfo {
         _layout = layout;
     }
     void set_dims(std::vector<size_t>& new_dims) {
-        if (_num_of_dims == new_dims.size()) {
-            _dims = new_dims;
-            modify_strides();
-            _data_size = _strides[0] * _dims[0];
-            set_max_shape();
-        } else {
-            THROW("The size of number of dimensions does not match with the dimensions of existing tensor")
-        }
+        if (_num_of_dims != new_dims.size())
+            WRN("The size of number of dimensions does not match with the dimensions of existing tensor")
```

Comment:
why this is changed from THROW to WRN?

Reply:
This was changed to handle the SKIP condition of missing component behaviour.